### PR TITLE
elvish 0.19.2

### PR DESCRIPTION
**Pull request by p-linnane**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by ZhongRuoyu** [comment]
Let's wait for upstream's response on the pre-release status before merging this: https://github.com/elves/elvish/pull/1666

**Comment by ZhongRuoyu** [comment]
As per upstream response in https://github.com/elves/elvish/pull/1666#issuecomment-1458893486, I suggest that we don't add the audit exception for this release, as we can always consider adding this next time. (IMO `all` is kinda too permissive and can be easily forgotten.)

**Review by ZhongRuoyu — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @ZhongRuoyu!

**Pull request closed without merging**
